src/systemd/sd-gpt.h

```diff
@@ -216,14 +216,14 @@ _SD_BEGIN_DECLARATIONS;
 #  define SD_GPT_USR_NATIVE_VERITY SD_GPT_USR_LOONGARCH64_VERITY
 #  define SD_GPT_USR_NATIVE_VERITY_SIG SD_GPT_USR_LOONGARCH64_VERITY_SIG
 
-#elif defined(__mips__) && __BYTE_ORDER__ == __ORDER_LITTLE_ENDIAN__
+#elif defined(__mips__) && !defined(__mips64) && __BYTE_ORDER__ == __ORDER_LITTLE_ENDIAN__
 #  define SD_GPT_ROOT_NATIVE SD_GPT_ROOT_MIPS_LE
 #  define SD_GPT_ROOT_NATIVE_VERITY SD_GPT_ROOT_MIPS_LE_VERITY
 #  define SD_GPT_ROOT_NATIVE_VERITY_SIG SD_GPT_ROOT_MIPS_LE_VERITY_SIG
 #  define SD_GPT_USR_NATIVE SD_GPT_USR_MIPS_LE
 #  define SD_GPT_USR_NATIVE_VERITY SD_GPT_USR_MIPS_LE_VERITY
 #  define SD_GPT_USR_NATIVE_VERITY_SIG SD_GPT_USR_MIPS_LE_VERITY_SIG
-#elif defined(__mips64__) && __BYTE_ORDER__ == __ORDER_LITTLE_ENDIAN__
+#elif defined(__mips64) && __BYTE_ORDER__ == __ORDER_LITTLE_ENDIAN__
```

Comment:
instead maybe swap the two blocks? do the `__mips64` defines first, and the `__mips__` defines second? sounds weird checking this twice

Reply:
if they get reordered for any reason it would break again, given this is exotic and I find out about breakages only when uploading to debian, better safe than sorry, it's just build time anyway